src/Settings/Attributes/Description.php

```diff
@@ -7,5 +7,15 @@
 #[Attribute(Attribute::TARGET_PROPERTY)]
 final class Description
 {
-    public function __construct(private string $description) {}
+    public function __construct(private readonly string $default, private readonly bool $required = true) {}
+
+    public function getDefault(): string
```

Comment:
I'm not a fan of prepending `get` on getters.

```suggestion
    public function default(): string
```